### PR TITLE
Add new lyrics page

### PR DESCRIPTION
**Pull request by JustJamie4realz**

### CODE REVIEW

**Comment by felix920506** [comment]
Please provide a description.

**Comment by nielsvanvelzen** [comment]
Why did you close the original pull request with a pending review...

File: docs/general/server/media/Lyrics.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+[00:24.86]Too many lovers she's scarred
+[00:26.82]And they want her back
+
+[00:29.50]She loves control
+[00:32.12]She wants it her way
+...
+```
+
+### Unsynchronized
+Unlike syncronized lyrics, unsyncronized lyrics are easier to implament, but will be harder for users to sing along.
```
Comment (Gauvino) [inline docs/general/server/media/Lyrics.md:45]:
```suggestion
Unlike synchronized lyrics, unsynchronized lyrics are easier to implement, but will be harder for users to sing along.
```

**Comment by M-Davies** [comment]
@JustJamie4realz has this been abandoned? Happy to open a new one if needs be

**Comment by felix920506** [comment]
@M-Davies just open a new PR please

**Comment by M-Davies** [comment]
> @M-Davies just open a new PR please

Opened https://github.com/jellyfin/jellyfin.org/pull/1133

**Comment by felix920506** [comment]
Closing in favor of #1133 since this thread looks abandoned

**Pull request closed without merging**